src/app/component/SliderWidget/SliderWidget.component.js

```diff
@@ -135,10 +137,18 @@ export class SliderWidget extends PureComponent {
         );
     };
 
+    renderPlaceholder() {
+        return <div block="WidgetFactory" elem="SliderPlaceholder" />;
+    }
```

Comment:
Why is this necessary here? This div does not have any styles, why not just `return null`?

Reply:
Reverted, back to `null`... due to bad local environment `return null` was not loading... after reinstall all is working.